nav_msgs/Path knot
float64[] knot_t
---
traj_gen/PolySplineXYZ spline_xyz 
